lms/djangoapps/courseware/access.py

```diff
@@ -173,53 +182,6 @@ def has_access(user, action, obj, course_key=None):
                     .format(type(obj)))
 
 
-# ================ Implementation helpers ================================
-
-def has_staff_access_to_preview_mode(user, obj, course_key=None):
```

Comment:
@noraiz-anwar you should test the functionality for live & preview modes to make sure that you're achieving acceptance criteria as expected. Also, you can take a look at this [PR](https://github.com/edx/edx-platform/pull/11953/files) to see if you're missing something.